config/Make.rules.Darwin

```diff
@@ -4,8 +4,10 @@
 
 supported-platforms             = macosx iphoneos iphonesimulator
 
+macos_arch                      ?= $(shell uname -m)
+
 macosx_ar                       = libtool
-macosx_cppflags                 = -mmacosx-version-min=10.14 -arch x86_64
+macosx_cppflags                 = -mmacosx-version-min=10.14 -arch $(macos_arch)
```

Comment:
Shouldn't we build a library with both Intel and ARM arch?

Reply:
We could, I don't have strong feelings either way.

Reply:
I'm going to remove it, I think we should just use the default. Should we do the same for the iphone target